### DISCUSSION TITLE
[RFC FS-1086 Discussion] - Public [<Literal>]s as static members

### DISCUSSION THREAD

**Discussion by Happypig375** [fsharp/fslang-design#469] [Language and core library RFC discussions]
Discussion for RFC FS-1086 - Public [\<Literal\>]s as static members

Suggestion here: fsharp/fslang-suggestions#746

**Comment by abelbraaksma** [comment]
Let's add a link to the RFC under discussion: https://github.com/fsharp/fslang-design/blob/master/RFCs/FS-1086-Public-Literals-as-static-members.md

**Comment by abelbraaksma** [comment]
Should we consider local literals as well? I believe that's also currently not possible, and even though let bindings can be compiled as constants by the optimizer, and usually are, they cannot be used in match expressions afaik.

Not sure this is out of scope.
